Lib/test/test_typing.py

```diff
@@ -4005,6 +4005,15 @@ def test_missing__name__(self):
                 )
         exec(code, {})
 
+    def test_subclassing(self):
+        with self.assertRaisesRegex(
+            TypeError, 
+            f"You cannot subclass an instance of NewType, perhaps you were looking for:\n"
```

Comment:
Let's use the same message we use for other types: https://github.com/python/cpython/blob/d0b0ec758cba80c38cdb78126d04314fdbe76c40/Lib/typing.py#L1100

Maybe with an extra note about `ProUserId = NewType('ProUserId', UserId)`

Reply:
```suggestion
            f"Cannot subclass {self!r}, perhaps you were looking for:\n"
```
Would this be better?

Reply:
I like it 👍